package.json

```diff
@@ -34,6 +34,10 @@
     "lint": "yarn workspaces foreach -A -p run lint && yarn prettier --write .github/*",
     "lint:check": "yarn workspaces foreach -A -p run lint:check && yarn prettier --check .github/*",
     "docs": "typedoc",
-    "gen:proto": "make -f ./scripts/compile_protos.mk"
+    "gen:proto": "make -f ./scripts/compile_protos.mk",
+    "postinstall": "patch-package"
```

Comment:
**issue:** Is it possible to have it in package.json for `examples/fishjam-chat`? It's not something that concerns our SDK 🤔

Reply:
I tried adding it to `examples/fishjam-chat` but unfortunately it does not work there. The `node_modules` are in the root directory and `patch-package` can't find them from there. I didn't find a way to force it to look for that elsewhere but I'm open for suggestions

Reply:
What do you think about this feature? 🤔 
https://yarnpkg.com/cli/patch